app/models/manageiq/providers/openshift/container_manager/refresher.rb

```diff
@@ -20,6 +32,7 @@ def parse_legacy_inventory(ems)
           fetch_entities(openshift_client, OPENSHIFT_ENTITIES)
         end
         entities = openshift_entities.merge(kube_entities)
+        entities[:metrics] = fetch_ems_metrics(ems) || {}
```

Comment:
see above comment about naming the functions `fetch_ems_metrics`

you do not collect metrics [ e.g. cpu_usage, filesystem_usage, network_rate .... ] you collect inventory [ e.g. server versions ] calling it metrics is confusing in this context.